templates/docs/examples/patterns/navigation/search-dark-new.html

```diff
@@ -9,9 +9,12 @@
 <header id="navigation" class="p-navigation is-dark">
   <div class="p-navigation__row">
     <div class="p-navigation__banner">
-      <div class="p-navigation__logo">
-        <a class="p-navigation__item" href="#">
-          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/3c7954dd-logo-canonical-white.svg" alt="" width="95" />
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="#">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
```

Comment:
The Circle of Friends is going to be a symbol for the Canonical company too?!

Reply:
It is a placeholder for now. But as far as I understand some smaller websites/products that don't have their own logo may use this instead when new branding is rolled out.